06_gpu_and_ml/text-to-audio/chatterbox_tts.py

```diff
@@ -0,0 +1,98 @@
+# ---
+# output-directory: "/tmp/chatterbox-tts"
+# lambda-test: false
```

Comment:
Should be at least smoke-testable with `modal serve` -- something like `cmd: ["modal", "serve", "path/from/root/whatever.py"]`.